packages/sources/src/sourceFormRenderer/components/CardSelect.js

```diff
@@ -37,7 +37,8 @@ class CardSelect extends React.Component {
     }
 
     handleKeyPress = (event, value) => {
-        if (event.key === 'Enter') {
+        if (event.key === ' ') {
```

Comment:
Is it possible to change this to use key code? `'  '` is easily overlooked I think.

Reply:
Fixed